std_msgs/Header header
uint8 stereo_cameras_direction
float64[9] left_intrinsics
float64[9] right_intrinsics
float64[9] rotation_left_in_right
float64[3] translation_left_in_right